soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1700,20 +1704,6 @@ void Randomizer::ParseRandomizerSettingsFile(const char* spoilerFileName) {
                             gSaveContext.randoSettings[index].value = 5;
```

Comment:
Why is this value set to 5? Is it neccesary for this option? Won't this break the cases right above this if it needs to be 5?

Reply:
the break is on line 1691, i think that's just github not knowing what to collapse

Reply:
Oh wow that's a VERY weird diff from github. Alright, looks good then 👍